### PR TITLE
s22-4pm-1 Added favicon, navbar icon, and footer copyright info

### LINKED ISSUE #48: Deploy Favicon With Storke Tower Image

### PR DESCRIPTION
**Pull request by AvivSamet**
# Overview

Added favicon, added the storke tower icon to the nav bar, and added the copyright text for the storke tower icon to the footer. Tests for the footer were also implemented. There is currently a bug with the image not rendering in the navbar when the backend is deployed.

# Issues Addressed

Closes #48 

# Link to Storybook
[Navbar Storybook](https://ucsb-cs156-s22.github.io/s22-4pm-courses-docs-qa/storybook-qa/AvivSamet-add-icon/?path=/story/components-nav-appnavbar--basic-logged-in-admin-user)

### CODE REVIEW

**Comment by AvivSamet** [comment]
I'm not sure why the production CIs are failing. However, I tried making the changes the eslint CI was asking for and the footer tests failed.

**Comment by roberto-garcia-0** [comment]
I am not sure if this bug will be reflected on the deployment. Will have to check that.

**Review by roberto-garcia-0 — APPROVED** [review]
Verified that the qa deployment did not have the same bug as in the localhost deployment.

Screenshot_20220602_151423

LGTM!

**Review by andrewhlu — DISMISSED** [review]
Tested on Heroku and changes look good! Some clarifying questions before I merge:

1. I saw this comment in your PR description:

> There is currently a bug with the image not rendering in the navbar when the backend is deployed.

Could you elaborate on what this means / provide steps to reproduce this issue? Additionally, is this filed as an actual issue in GitHub so future iterations can work on this?

2. I saw that you had added Storybook support for the navbar, but I don't see a new story for the new footer. Did you plan on adding those here?

**Comment by AvivSamet** [comment]
> Tested on Heroku and changes look good! Some clarifying questions before I merge:
> 
> 1. I saw this comment in your PR description:
> 
> > There is currently a bug with the image not rendering in the navbar when the backend is deployed.
> 
> Could you elaborate on what this means / provide steps to reproduce this issue? Additionally, is this filed as an actual issue in GitHub so future iterations can work on this?
> 
> 2. I saw that you had added Storybook support for the navbar, but I don't see a new story for the new footer. Did you plan on adding those here?

Bryan just told me that the image not showing on localhost:8080 when the backend is deployed was a known issue. I'm not sure what I can do to fix it. I wasn't planning on adding a storybook for the footer.

**Review by roberto-garcia-0 — APPROVED** [review]
LGTM!

File: frontend/src/main/components/Nav/Footer.js [hunk 1]
```diff
@@ -1,12 +1,60 @@
...
+      .
+      </p>
+      <p>
+        The cartoon Storke Tower images in the brand logo and favicon for this site were
+        developed by Chelsea Lyon-Hayden, Art Director for UCSB Associate Students, and are
+        used here by permission of the Executive Director of UCSB Associated Students.
+        These images are Copyright © 2021 UCSB Associated Students, and may not be reused
+        without express written permission of the Executive Director of UCSB Associated Students.  For more info, visit: 
+        {space}
+        <a data-testid="footer-sticker-link" href="https://www.as.ucsb.edu/sticker-packs/>">www.as.ucsb.edu/sticker-packs/</a>
```
Comment (pconrad) [inline frontend/src/main/components/Nav/Footer.js:56]: Check the syntax on this link...

**Review by pconrad — CHANGES_REQUESTED** [review]
One small fix... the link to the sticker pack has a syntax error I think.

**Review by roberto-garcia-0 — APPROVED** [review]

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**